common.gradle

```diff
@@ -0,0 +1,27 @@
+//version=5.36
+ext{
+    buildNum=0
```

Comment:
buildNum was/is something that Ant tracks, Gradle does not.  If this build number is important then steps will need to be taken to use, iterates and track this build number.